zulipterminal/ui_tools/boxes.py

```diff
@@ -170,6 +173,15 @@ def need_recipient_header(self) -> bool:
         else:
             raise RuntimeError("Invalid message type")
 
+    def _check_if_self_PM(self) -> bool:
```

Comment:
This is for the bug fix when you sent a PM to yourself. If you wanted to reply to a message( mouse clicking on it / `r`/ ENTER), a fotter is created. The email address in 'To:` would be blank for such PM's with oneself.